apps/next/tsconfig.app.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
-    "types": []
+    "types": ["css-modules"]
```

Comment:
I guess we don't need this one

Reply:
We need this in order for the next app build to succeed

Reply:
He is correct, we can remov ethis once #1291 is merged, but until then, it is still required.